components/skeleton/demo/index.html

```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<meta http-equiv="X-UA-Compatible" content="ie=edge">
+	<meta charset="UTF-8">
+	<link rel="stylesheet" href="../../demo/styles.css" type="text/css">
+	<script src="/node_modules/@webcomponents/webcomponentsjs/webcomponents-loader.js"></script>
+	<script type="module">
+		import '../../demo/demo-page.js';
+		import '../../button/button.js';
+		import '../../inputs/input-text.js';
+		import '../../inputs/input-date.js';
+		import './skeleton-host.js';
+	</script>
+	<style>
+		@keyframes loadingPulse {
```

Comment:
For the demo, I just copied the skeleton styles to the demo page as didn't want to spend time making them work both in a component host and here. But ordinarily these would only be needed within a host using the the `skeleton-styles.js` style module.